### ISSUE TITLE
Linenos and minification

### ISSUE THREAD

**Issue by pcoves** [Keats/gutenberg#1743]
# Bug Report

## Environment

Zola version: `0.15` (already in `0.14`)

## Expected Behavior

Multiple *fenced blocks* with `linenos` should coexists with `minify_html = true`.

## Current Behavior

If there are multiple *fenced blocks* with `linenos`, the *code block* spread from first to last.
Meaning, it embeds everything on its path, including non-code-block content.

### `config.toml`

Enable highlighting and minification.

```
base_url = "https://example.com"
compile_sass = true
build_search_index = false
minify_html = true

[markdown]
highlight_code = true

[extra]
```
### `templates/index.html`

```html
<html>
    <body>
        {{ section.content | safe }}
    </body>
</html>
```

### `content/_index.md`

    +++
    +++

    ```linenos
    One line: no issue
    Two or more lines: issue
    ```

    Foo
    Bar
    Baz

    ```linenos
    Autruche
    ```
![2022-01-24-145543_254x125_scrot](https://user-images.githubusercontent.com/33952527/150795732-7799015c-d82f-4711-baf8-8523940b7cee.png)

![2022-01-24-145617_284x195_scrot](https://user-images.githubusercontent.com/33952527/150795810-3c4c73dd-c9ea-48e7-9137-6a9fa0acb749.png)

**Comment by mwcz** [comment]
There were some unclosed `<tr>` and `<td>` tags for multi-line fenced code blocks in `components/rendering/src/codeblock/mod.rs`.  PR incoming.

**Comment by welpo** [comment]
Is the fix live as of `0.17.2`?

I'm having this issue of "unclosed tables", even with a single blockcode with `linenos`.

The issue disappears when I set `minify_html` to `false`.

**Comment by Keats** [comment]
It's live yes. There is another issue with class-based block formatters though: https://github.com/getzola/zola/pull/2237 and the related issue.
